qodana.yaml

```
@@ -0,0 +1,35 @@
+version: "1.0"
+linter: qodana-clang
+bootstrap:  sh ./scripts/prepare_qodana.sh
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)